public/app/core/components/AppChrome/AppChromeService.tsx

```diff
@@ -177,9 +177,9 @@ export class AppChromeService {
     return useObservable(this.state, this.state.getValue());
   }
 
-  public setMegaMenuOpen = (newOpenState: boolean) => {
+  public setMegaMenuOpen = (newOpenState: boolean, updatePersistedState = true) => {
```

Comment:
Shouldn't this be `updatePersistedState = false`?

Reply:
No I can't think why it'd default to false? We want the default to match the existing behaviour, which is to persist the change?